apps/voting/app/src/utils-hooks.js

```diff
@@ -71,3 +71,9 @@ export function usePanelState({ onDidOpen = noop, onDidClose = noop } = {}) {
     [requestOpen, requestClose, visible, didOpen, onTransitionEnd]
   )
 }
+
+export function useScrollTop(dependency) {
```

Comment:
I find this type of files: `utils-x` become a bag of cats down the road, by naming files for hooks (like we do components) we can group together behaviour and dependencies that go together. Maybe use a `hook-utils` or `hooks` folder with the hooks in it?

Reply:
A `hooks` folder sounds good; we may also eventually want to have a `utils` folder to organize the root a bit more.

Reply:
Done in 9e1222e.